.circleci/run.sh

```diff
@@ -194,8 +194,7 @@ codecov()
     # CodeCov gets confused by lst files which it can't matched
     rm -rf test/runnable/extra-files
     download "https://codecov.io/bash" "https://raw.githubusercontent.com/codecov/codecov-bash/master/codecov" "codecov.sh"
-    cd src # need to run from compilation folder for gcov to find sources
-    bash ../codecov.sh -p .. -x gcov-4.9 # must match g++ version (on CircleCI `g++` is 4.9 and `gcov` 4.6 :/)
```

Comment:
What's the story here ?

Reply:
Changing the working directory implies different paths for coverage files.
`dmd_expressionsem.lst` located in `src` becomes `src_dmd_expressionsem.d` in the root directory.

See https://github.com/dlang/dmd/pull/11534#discussion_r467500690 regarding `gcov`